src/analyzer/protocol/websocket/unmask.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024 by the Zeek Project. See COPYING for details.
+
+// Import HILTI
+#include <hilti/rt/libhilti.h>
+
+namespace hlt_websocket::WebSocket {
+
+// Implement XOR unmasking of WebSocket frames due to being very
+// hot and inefficient in pure Spicy.
```

Comment:
The comment is unnecessarily judgemental (e.g., I'd expect that being able to write that function in Spicy code was an efficient use of your time, but you particularly care about throughput here).

```suggestion
// Implement XOR unmasking of WebSocket frames in C++ since this code is very hot.
```